cds/config.py

```diff
@@ -114,6 +117,8 @@ def _(x):
     'APP_CACHE_REDIS_URL',
     'redis://localhost:6379/0')
 CACHE_TYPE = 'redis'
+CACHE_IS_AUTHENTICATED_CALLBACK = lambda: '_flashes' in session or \
```

Comment:
Nice! does it work correctly?

I don't know if we prefer to keep the decorator or use this lambda, if terms of code readability... @egabancho can better answer this!

Reply:
Actually there was no point to keep `decorators.py` as there was no decorator left inside. But I don't know if we have any other place to put this function that's why I just created a lambda. I could even write a small function inside the `config.py` if we need more readability.